app/src/main/java/com/example/mycalculator/mvp/model/CalculatorModel.java

```diff
@@ -37,7 +37,10 @@ public String getFullResult() {
         switch (operator) {
 
             case OPERATOR_SUM:
-                result = String.valueOf((Double.parseDouble(firstOperand) + Double.parseDouble(secondOperand)));
+                if(result == EMPTY_STRING){
```

Comment:
another way to check if a value is equals to " "  , is using this method ".isEmpty"

Reply:
Thx for the advice.